src/libraries/System.Net.HttpListener/tests/TrimmingTests/System.Net.HttpListener.TrimmingTests.proj

```diff
@@ -1,6 +1,10 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
 
+  <PropertyGroup>
+    <SkipOnTestRuntimes>browser-wasm</SkipOnTestRuntimes> <!-- Justification: HttpListener.ctor() throws PNSE on wasm -->
```

Comment:
Do you have a stack trace for that? We don't have that API annotated as unsupported on the browser

@buyaa-n any thoughts about why the analyzer didn't catch use of ExtendedProtectionPolicy ?

Reply:
Take a look at this:

https://github.com/dotnet/runtime/blob/d99ddd0699dc209714ca8d5e9ff1f5706604911a/src/libraries/System.Net.HttpListener/src/System.Net.HttpListener.csproj#L7

the whole assembly will throw PNSE as the only two implementations that actually don't throw are the windows and linux specific, all of the rest of the platforms throw.

Reply:
> @buyaa-n any thoughts about why the analyzer didn't catch use of ExtendedProtectionPolicy ?

Not sure, could you point out where that `ExtendedProtectionPolicy` is used? I see `System.Net.HttpListener` has assembly level `UnsupportedOSPlatforms("browser")` attribute, https://github.com/dotnet/runtime/blob/8a3fd5a065b019cc70a600a759ea0c87c14cae8f/src/libraries/System.Net.HttpListener/Directory.Build.props#L6 if `ExtendedProtectionPolicy` is API from  `System.Net.HttpListener` it should warn for its usage

Reply:
Oh, I missed that line.